src/include/miopen/execution_context.hpp

```diff
@@ -85,9 +85,14 @@ struct ExecutionContext
     inline Handle& GetStream() const { return *stream; }
     inline void SetStream(Handle* stream_) { stream = stream_; }
 
-    ExecutionContext() = default;
     ExecutionContext(Handle* stream_) : stream(stream_) {}
-    virtual ~ExecutionContext() = default;
+
+    ExecutionContext() {}
```

Comment:
Does this produce a warning with `= default` and other changes?

Reply:
I have pushed this docker: `rocm/miopen:debug_2063` and so far no warning messages.